packages/cli-kit/src/public/node/api/admin.ts

```diff
@@ -18,7 +19,7 @@ export async function adminRequest<T>(query: string, session: AdminSession, vari
   const api = 'Admin'
   const version = await fetchApiVersion(session)
   const url = adminUrl(session.storeFqdn, version)
-  return graphqlRequest(query, api, url, session.token, variables)
+  return graphqlRequest(query, api, url, buildHeaders(session.token), variables)
```

Comment:
All graphql calls should include the default headers, so I think it'd be better for this to be inside `graphqlRequest`.
Is there a reason why oxygen wouldn't use `buildHeaders`?

Reply:
Oxygen-hub requires a header `X-Oxygen-Namespace-Name` which contains the namespace for the app that's deploying. For Hydrogen storefronts that would be a static value, but that may change should we allow non-storefront deployments. I think it's best if we have control over this header from our plugin.

Reply:
that makes perfect sense. I've added an optional `addedHeaders` argument 👍